sympy/printing/repr.py

```diff
@@ -144,6 +144,14 @@ def _print_EmptySequence(self, expr):
     def _print_list(self, expr):
         return "[%s]" % self.reprify(expr, ", ")
 
+    def _print_dict(self, expr):
+        sep = ", "
+        dict_kvs = ["%s: %s" % (self.doprint(key), self.doprint(value)) for key, value in expr.items()]
```

Comment:
Most of the other implementations use `_print` I think
```suggestion
        dict_kvs = ["%s: %s" % (self._print(key), self._print(value)) for key, value in expr.items()]
```

Reply:
Although I suppose `reprify` does not, so this doesn't really matter.